src/components/NotesSection/NotesDisplay/NotesDisplay.jsx

```diff
@@ -0,0 +1,84 @@
+import { useNotes } from "../../../contexts";
+import NotesCard from "./NotesCard/NotesCard";
+
+import "./notesDisplay.css";
+
+const NotesDisplay = ({ inTrash }) => {
+  const { notes } = useNotes();
+  let mappedNotes = notes;
+
+  if (inTrash) {
+    mappedNotes = notes.filter((note) => note.isTrashed);
+  }
+  const pinnedNotes = mappedNotes.filter(
+    (note) => note.isPinned && !note.isTrashed
+  );
+  const otherNotes = mappedNotes.filter(
+    (note) => !note.isPinned && !note.isTrashed
+  );
+
+  return pinnedNotes.length === 0 ? (
+    <div className="container-notes">
+      {inTrash
+        ? mappedNotes.map((item) => (
+            <NotesCard
+              key={`Note${item.id}`}
+              noteId={item.id}
+              tagId={item.id}

```

Comment:
I think you can use rest operator instead of these may props. Code might look better